README.md

```diff
@@ -141,30 +151,72 @@ fn main() -> pgp::errors::Result<()> {
 - Security Status: [STATUS_SECURITY.md](docs/SECURITY_STATUS.md)
 - Supported Platforms: [PLATFORMS.md](docs/PLATFORMS.md)
 
-## rPGP for application developers
+## FAQs
 
-rPGP aims to make it easy for application developers to incorporate OpenPGP functionality into their projects.
+Checkout [FAQ.md](docs/FAQ.md).
 
-However, the OpenPGP format and its semantics are relatively complex, and there has historically not been a reader-friendly introduction to the domain.
+## OpenPGP Versions and Features
 
-The text ["OpenPGP for application developers"](https://openpgp.dev/) offers an introduction to OpenPGP, aimed mainly at application developers. We recommend it for initial orientation.
+Checkout [openpgp.md](docs/openpgp.md).
 
-Independently, we welcome questions in the rPGP issue tracker.
+## rPGP is a low-level OpenPGP library
```

Comment:
it is very confusing that this comes after a bunch of links to docs, with such a long paragraph